### DISCUSSION TITLE
Localhost Secret Key?

### DISCUSSION THREAD

**Discussion by smailschweizer** [webstudio-is/webstudio-community#162] [General]
Hi there,

I'm trying to install Webstudio Editor locally on my Mac. I need to work without internet… Most of it worked, but now i need a secret key to log in. Where do i find it? How do i generate it? Surry, big noobie here ;))

<img width="362" alt="Bildschirmfoto 2024-07-13 um 07 09 08" src="https://github.com/user-attachments/assets/84253dfe-ab68-4c98-b12e-1e8cca15d4c3">

**Comment by kof** [comment]
we updated the docs, you need to put it into .env.development

**Reply by kof** [reply]
https://docs.webstudio.is/contributing/contributing-for-developers#dev-login
